apps/web/src/hooks/playgroundChat/useProviderEventHandler.ts

```diff
@@ -376,22 +236,9 @@ export function useProviderEventHandler({
         case 'tool-result':
           handleToolResult(data)
           break
-        case 'reasoning':
```

Comment:
QA reasoning keep working

Reply:
it workz but the FE remains in "thinking" state even after the streaming has ended 
<img width="773" height="274" alt="image" src="https://github.com/user-attachments/assets/c31d34d7-1816-409f-82ea-8e5391355608" />